src-windows/Lukko/Windows.hsc

```diff
@@ -134,31 +134,16 @@ unlockImpl :: FD -> IO ()
 unlockImpl (FD wh) = do
   allocaBytes sizeof_OVERLAPPED $ \ovrlpd -> do
     fillBytes ovrlpd 0 sizeof_OVERLAPPED
-    c_UnlockFileEx wh 0 0xffffffff 0xffffffff ovrlpd >>= \res -> case res of
+    c_UnlockFileEx wh 0 #{const INFINITE} #{const INFINITE} ovrlpd >>= \res -> case res of
       True  -> return ()
       False -> getLastError >>= failWith "fdUnlock"
   where
     sizeof_OVERLAPPED = #{size OVERLAPPED}
-#if defined(i386_HOST_ARCH)
 
 -- https://docs.microsoft.com/en-gb/windows/win32/api/fileapi/nf-fileapi-lockfileex
-foreign import stdcall interruptible "LockFileEx"
+foreign import WINDOWS_CCONV interruptible "LockFileEx"
```

Comment:
This module already defines `WINDOWS_CCONV` in https://github.com/phadej/lukko/pull/13/files#diff-4f4793caa24ce17d8fe75149ab491696L45-L51 -- so we might as well use it...

Reply:
It actually doesn't work, in `hsc` files. Rather we shopuld remove defines which are left overs of an experiment.

Reply:
yeah, I forgot to protect the `#defines` so they make it into the 2nd CPP phase... fixed now